Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -166,3 +191,19 @@ extension ExperimentalAnimationLayer: RootAnimationLayer {
   }
 
 }
+
+// MARK: - CALayer + allSublayers
+
+extension CALayer {
+  /// All of the layers in the layer tree that are descendants from this later
+  @nonobjc
+  fileprivate var allSublayers: [CALayer] {
+    var allSublayers: [CALayer] = []
+
+    for sublayer in sublayers ?? [] {
+      allSublayers += [sublayer] + sublayer.allSublayers
```

Comment:
nit: seems like this will create less throwaway arrays
```suggestion
      allSublayers.append(sublayer)
      allSublayers.append(contentsOf: sublayer.allSublayers)
```

Reply:
good point, hadn't thought of that